history-of-the-quakers/original/19-book-2-chapter-10.adoc

```diff
@@ -146,16 +165,19 @@ in men by the possession of power,
 according to the concurrent accounts historians have
 given us of the independents and their principles:
 
-"`The fanaticism of the independents abolished all ecclesiastical government,
+[.embedded-content-document]
```

Comment:
Hmm... I think I would prefer this to be a `[quote]` not an embed. My thought process is, wherever Gough is sort of quoting a chunk from a history book or something like that, it should probably be a quote, not an embed. The embeds should be more documents that are written by people involved in what is being described, and at about the same time.  So letters from Fox, Nayler, etc., legal documents to/from parlaimant, arrest warrants, epistles, etc. should all be embeds -- but then when Gough switches to sort of using other books as references, and quotes them at length, make them a [quote].  hopefully that distinction makes sense!

Reply:
Yes, it does!! Thanks.